### PR TITLE
Rewrite vim-gitgutter suggestion in README.md

### PR DESCRIPTION
**Pull request by kutsan**
I think signify shouldn't underrate itself. In my honest opinion signify is better than gitgutter. It's just a better product and users should consider using this plugin even if git is the only version control system they use.

When I first saw that statement I thought even the author of this plugin suggests using gitgutter, then gitgutter must be better than signify. I believe newcomers think like that too.

Long story short, I propose to remove that sentence altogether.

### CODE REVIEW

**Comment by mhinz** [comment]
Hey,

I'm glad you like the plugin!

Keeping the message would only be fair considering that vim-gitgutter links vim-signify from their README as well.

But we can talk about the wording. As far as I see (from their README), the only real advantage is the staging support?

What about making it very short?

_Alternative: [vim-gitgutter](https://github.com/airblade/vim-gitgutter)_

Then people can read their README and decide for themselves.

**Comment by kutsan** [comment]
> Keeping the message would only be fair considering that vim-gitgutter links vim-signify from their README as well.

_"Supports git only. If you work with other version control systems, I recommend vim-signify."_

They recommend signify because they can't work with version control systems other than git. Majority of us use git only. So, there isn't any promotion at all. If someone is already using gitgutter and saw that statement in their README they wouldn't even bother clicking because they only use git. It's rather bad because users don't think signify as an alternative but an another project for not-git-people. It's not the same with signify's README. Semantics, I'd say.

_"If git is the only version control system you use, I suggest having a look at vim-gitgutter."_

In this case, although signify supports git very well, you still send your users to another plugin project. Do they deserve this kind of promotion? I think no. Also, your statement is very visible in README and eye-catching.

> What about making it very short? Then people can read their README and decide for themselves.

If you still not okay to remove it completely, where do you think should we put that statement? There isn't much space in README.

**Comment by mhinz** [comment]
It's just a plugin. ;-)

I care about the users of the plugin, not about users who got driven away by that sentence. I don't get paid for it, so there's no need to maximise the user count.

I'll accept the PR if the sentence gets changed to:

_Similar plugin for git: [vim-gitgutter](https://github.com/airblade/vim-gitgutter)_

Otherwise I'd like to spend my time on other things. :>

**Comment by kutsan** [comment]
Sure, have a look at the commit.
